app/src/main/java/com/example/foodmanchu/IngredientsFragment.kt

```diff
@@ -2,29 +2,66 @@ package com.example.foodmanchu
 
 import android.os.Bundle
 import android.view.View
+import android.widget.Adapter
+import android.widget.Toast
 import androidx.fragment.app.Fragment
 import androidx.recyclerview.widget.LinearLayoutManager
 import com.example.foodmanchu.databinding.IngredientsFragmentBinding
 
 class IngredientsFragment:Fragment(R.layout.ingredients_fragment) {
+
+    private var ingredientsAdapter: IngredientsAdapter ?= null
+
```

Comment:
Just a thought: This could be a lateinit, and get rid of a null.
```suggestion

    private var ingredientsAdapter: IngredientsAdapter ?= null

```

Reply:
testing if comments exist after resolving

Reply:
changed to late init, better

Reply:
leaving this one alive, I resolved the other ones, is that ok, or should be done on your side after correcting